doc/flatpak-install.xml

```diff
@@ -78,10 +78,10 @@
         </para>
         <para>
           If the specified <arg choice="plain">REMOTE</arg> has a collection ID configured on it,
-          flatpak will search mounted filesystems such as USB drives as well as Avahi services
```

Comment:
Is the Avahi reference lost, I don't see it in other man pages?

Reply:
I'm not sure what you mean by lost? We used to support finding peers over Avahi, but don't any more.

Reply:
Oh, OK.